h/search/index.py

```diff
@@ -74,19 +75,18 @@ def index(self, annotation_ids=None, windowsize=PG_WINDOW_SIZE):
         return errored
 
     def _prepare(self, annotation):
-        action = {
-            self.op_type: {
-                "_index": self._target_index,
-                "_type": self.es_client.mapping_type,
-                "_id": annotation.id,
-            }
+        operation = {
+            "_index": self._target_index,
+            "_id": annotation.id,
         }
+        if self.es_client.server_version < Version("7.0.0"):
+            operation["_type"] = self.es_client.mapping_type
```

Comment:
Slight re-ordering here so we don't repeat the `self.op_type` reaching into the dict